fabischolasi/fast-food-fast-v1
# #160799768 menus api endpoints

**What does this PR do?**
  - This pr contains api endpoints to manipulate menu

**How should this be manually tested?**
  - clone this branch, make sure postman is present and then add some functions to run your local server
  - go to postman and put the right url and test if the endpoints are working
  - make sure you have postgresql installed

**Any background context you want to provide?**
  - use of flask_restful to create http verbs

**What are the relevant pivotal tracker stories?**
  - #160799768
